test/sample-test/utils.py

```diff
@@ -88,15 +89,17 @@ def file_injection(file_in, tmp_file_out, subs):
 
 def upload_blob(bucket_name, source_file_name, destination_blob_name):
   """Uploads a file to the bucket."""
+  storage_client = storage.Client()
   try:
-    storage_client = storage.Client()
     bucket = storage_client.get_bucket(bucket_name)
-    blob = bucket.blob(destination_blob_name)
+  except google.cloud.exceptions.NotFound as e:
+    raise RuntimeError("Bucket not found: {}\n".format(str(e)))
 
+  blob = bucket.blob(destination_blob_name)
+  try:
     blob.upload_from_filename(source_file_name)
-  except:
-    raise RuntimeError('Failure when uploading %s from %s to %s'
-                       % (source_file_name, bucket_name, destination_blob_name))
+  except google.cloud.exceptions.GoogleCloudError as e:
```

Comment:
FYI, except's can be combined as such:
try:
except A:
except B:

Reply:
Good call. Done.